runtime/runtime-params-estimator/src/vm_estimator.rs

```diff
@@ -105,22 +105,22 @@ pub fn cost_per_op(gas_metric: GasMetric, code: &ContractCode) -> Ratio<u64> {
     //    (measured * vm_config.regular_op_cost) /
     //       (outcome.burnt_gas * NUM_ITERATIONS),
     // as remaining can be computed with ratio_to_gas().
-    Ratio::new(measured * (vm_config.regular_op_cost as u64), NUM_ITERATIONS * outcome.burnt_gas)
+    measured * vm_config.regular_op_cost as u64 / NUM_ITERATIONS / outcome.burnt_gas
 }
 
 type CompileCost = (u64, Ratio<u64>);
```

Comment:
> Can we change Ratio<u64> to Gas here?
I think yes, `Gas` makes sense here. And we should probably also use the raw instruction counter, without factoring in the IO bytes. 
I guess I haven't looked very closely while refactoring. I just made sure the function does what it did before.
But tbh, I still don't quite understand how and in what context we use this function.

Reply:
> More generally, I think we might want to move ad-hoc functions from here to general infra in `lib.rs`, though I need to take a closer look to learn specifics.

Agreed, it seems cleaner. But I would argue that should go in a different PR.

Reply:
Without moving any code to different files yet, I applied the changes to the function that think make sense. But it is a bit of guess work, since I don't know how we even use this.